lib/Doctrine/DBAL/Types/Exception/SerializationFailed.php

```diff
@@ -21,7 +21,7 @@ public static function new($value, string $format, string $error) : self
 
         return new self(
             sprintf(
-                "Could not convert PHP type '%s' to '%s', as an '%s' error was triggered by the serialization",
+                'Could not convert PHP type "%s" to "%s", as an "%s" error was triggered by the serialization',
```

Comment:
Can we move the chained error message the end, similarly to `ValueNotConvertible`?